applications/migrations/0014_gender_other.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.4 on 2019-02-11 22:15
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
```

Comment:
Should `dependencies` be empty? I think this is autopopulated by the migration generator.

Reply:
No, it needs to have the previous one in order to work. Used to keep in DB what migrations have been applied.